app/services/analytics_events.rb

```diff
@@ -3208,13 +3208,15 @@ def idv_sdk_error_before_init(
   # @param [String] acuant_version
   # @param [Integer] captureAttempts number of attempts to capture / upload an image
   #                  (previously called "attempt")
+  # @param [Integer] selfieAttempts number of times SDK captured selfie, user may decide to retake
   # rubocop:disable Naming/VariableName,Naming/MethodParameterName
   def idv_sdk_selfie_image_capture_closed_without_photo(acuant_version:, captureAttempts: nil,
-                                                        **extra)
+                                                        selfieAttempts: nil, **extra)
     track_event(
       :idv_sdk_selfie_image_capture_closed_without_photo,
       acuant_version: acuant_version,
       captureAttempts: captureAttempts,
+      selfieAttempts: selfieAttempts,
```

Comment:
Could we turn this into snake case (`selfie_attempts`) on the back end? I know it's not consistent in this file and it's a little tricky because we have to do camel case on the front end and then snake case on the back end. But it makes the linters happy and it's come up [in a few recent PRs](https://github.com/18F/identity-idp/pull/10368).

Reply:
@night-jellyfish , changed.